src/backend/helpers/pluginHelpers.ts

```diff
@@ -15,15 +16,14 @@ export const getInstalledPlugins = (
 export const insertInstalledPlugin = (
   supabase: SupabaseClient,
   project_id: string,
-  plugin_id: string,
   plugin_name: string
 ): Response<InstalledPlugin> =>
   supabase  
     .from('installed_plugins')
     .insert({
       project_id,
-      plugin_id,
-      plugin_name
+      plugin_name,
+      plugin_id: uuidv4() // Just a hack...
```

Comment:
Not sure if we need the plugin ID any more. It used to be an (arbitrary) UUID, which plugin developers were supposed to provide with their plugin. In hindsight, this feels odd from a developer perspective. 

With the new SDK, the plugin developer only declares a `name`. But perhaps we'd want a `name` and a `version`? (Both string fields.)

Reply:
@rsimon including version seems like a good idea! Questions: 
- Are plugins always npm packages, and installed via insertion into `package.json`?
- Should we require them to use [semver](https://semver.org/)?
- Should the version of the plugin always be the same as the version of the NPM package (I presume)?

Reply:
<Are plugins always npm packages, and installed via insertion into package.json?>

That's right. They are always npm packages, although in production we'd probably install them via the `add` utility command that comes with Astro. (Because that will install the package + make the changes to the Astro config.)

<Should we require them to use semver?>

Makes sense.

<Should the version of the plugin always be the same as the version of the NPM package (I presume)?>

Yes. In theory we could probably read from the package.json, although it would involve a little extra hacking.